### ISSUE TITLE
Add the breadcrumb navigation and header to the top of subscription page

### ISSUE THREAD

**Issue by wouter1975** [workfloworchestrator/orchestrator-ui#79]
- [ ] Add the breadcrumb navigation to the top of the page. Base the text of the breadcrumb on the navigation within the app.
- [ ] Add `Subscriptions` header 

![image](https://github.com/workfloworchestrator/orchestrator-ui/assets/83226590/ba2f6e11-dbb9-4767-a687-12b0206d07a8)

**Comment by acidjunk** [comment]
There is some ambiguity  in the design:

```
Start / Subscriptions / Active ->
```

- currently there is no "Active"
- A click on "Subscriptions" always goes to the first sub menu item
- When you navigate to a detail page suddenly you can't go back to "Active" beacuase it's not in the Crumble path.

So it doens't feel very consistent yet

This is from the design on detail page:

<img width="579" alt="Schermafbeelding 2023-06-10 om 13 49 52" src="https://github.com/workfloworchestrator/orchestrator-ui/assets/685002/190c6166-f4c4-49bd-ba51-219482552c39">

It has:

Start / Subscriptions / Subscription-description (and then repeat the description in the header)

But it should be (I think)

```
Start / Subscriptions / Active ->

Description Header```

**Comment by wouter1975** [comment]
Agreed, we need to finalise the choice of the breadbrumb. Active or Complete not a new page/table, but just another view on data. Therefore I would not include it in the breadcrumb, using the tab component should allow you to navigate to the other pages. Let's discuss in the next meeting.

**Comment by acidjunk** [comment]
Fixed in #61
